smart_proxy_realm_ad_plugin.gemspec

```
@@ -5,20 +5,24 @@ Gem::Specification.new do |s|
   s.name        = 'smart_proxy_realm_ad_plugin'
   s.version     = Proxy::AdRealm::VERSION
   s.date        = Date.today.to_s
-  s.license     = 'GPL-3.0'
+  s.license     = 'GPL-3.0-only'
   s.authors     = ['Mårten Cassel']
   s.email       = ['marten.cassel@gmail.com']
   s.homepage    = 'https://github.com/theforeman/smart_proxy_realm_ad_plugin'
+  s.required_ruby_version = '>= 2.7'
 
   s.summary     = "A realm ad provider plugin for Foreman's smart proxy"
-  s.description = "A realm ad provider plugin for Foreman's smart proxy"
+  s.description = "This plugin provides a realm ad provider for Foreman's smart proxy, allowing integration with Active Directory realms."
+
 
   s.files       = Dir['{config,lib,bundler.d}/**/*'] + ['README.md', 'LICENSE']
   s.test_files  = Dir['test/**/*']
 
-  s.add_development_dependency('rake')
-  s.add_development_dependency('mocha')
-  s.add_development_dependency('test-unit')
-  s.add_dependency('rkerberos')
-  s.add_dependency('radcli')
-end
+  s.add_development_dependency('rake', '~> 13.2.1')
```

Please don't pin to such exact versions. This prevents using rake 13.3 when it comes out. It's safe to assume rake respects semver and only pin to x.y. versions.
```suggestion
  s.add_development_dependency('rake', '~> 13.2')
```